CONTRIBUTING.md

```diff
@@ -144,13 +144,13 @@ First of all, you need the Zeppelin source code. The official location for Zeppe
 Get the source code on your development machine using git.
 
 ```
-git clone http://git.apache.org/incubator-zeppelin.git zeppelin
+git clone git://git.apache.org/incubator-zeppelin.git zeppelin
 ```
 
-You may also want to develop against a specific release. For example, for branch-0.1
+You may also want to develop against a specific release. For example, for branch-0.5.6
```

Comment:
Precisely, releases are available through tag in `v[X].[Y].[Z]` format. Let's change sentence from `... against a specific release. ...` to `... against a specific branch. ```